users/migrations/0009_auto_20190724_1749.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.0.6 on 2019-07-24 14:49
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('users', '0008_auto_20190724_1749'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='user',
+            name='gender',
+            field=models.CharField(choices=[('MALE', 'Male'), ('FEMALE', 'Female')], default='MALE', max_length=6),
+        ),
```

Comment:
You created too much migrations, this may increase your downtime in production. Let's use 1 migration for this changes.
1 - Roll back your database to migration 6.
2 - Delete migrations 7,8,9.
3 - Create migration 7.
4 - Run migration 7.